pkg/cluster/fixmcsuserdata_test.go

```diff
@@ -184,3 +192,76 @@ func TestFixMCSUserData(t *testing.T) {
 		}
 	}
 }
+
+func TestGetUserDataSecretReference(t *testing.T) {
```

Comment:
@yehornaumenko you will soon be looking for a new task - here is one for you :)

Could you please look into expanding (in a separate PR) this test to provide better coverage for `getUserDataSecretReference`?

![image](https://user-images.githubusercontent.com/509198/177984821-9e979e54-3ba0-474c-8100-611439240cf5.png)